exercises/concept/squeaky-clean/.meta/src/reference/java/SqueakyClean.java

```diff
@@ -1,11 +1,24 @@
+import java.util.Map;
+import java.util.HashMap;
+
 class SqueakyClean {
+    private static Map<Character, Character> leetSpeakMap = new HashMap<>() {{
+            put('3', 'e');
+            put('4', 'a');
+            put('0', 'o');
+            put('1', 'l');
+            put('7', 't');
+        }};
```

Comment:
The only concern I have here is that this file is considered to be the "exemplar" solution for this exercise. It is shown to mentors with the remark that they should steer students towards an equivalent solution.

Using a `Map` here effectively means that the `maps` concept (which doesn't exist yet btw) becomes a prerequisite for this exercise. This would be a little much IMO.

Can we maybe dumb down the exemplar solution a bit by converting the map to a helper method instead? Something like this:

```java
class SqueakyClean {
    private static char replaceDigit(char digit) {
        if (digit == '3') return 'e';
        if (digit == '4') return 'a';
        // etc
    }
}
```

Reply:
Done!